subsys/net/lib/nrf_cloud/Kconfig.nrf_cloud_pgps

```diff
@@ -5,6 +5,7 @@
 
 menuconfig NRF_CLOUD_PGPS
 	bool "Enable nRF Cloud Predicted GPS (P-GPS)"
+	depends on NRF_CLOUD_REST || NRF_CLOUD_MQTT
```

Comment:
I guess either one should be present, right @plskeggs?

Reply:
yes